Source/WebKit/WebProcess/WebPage/EventDispatcher.messages.in

```diff
@@ -26,7 +26,7 @@ messages -> EventDispatcher NotRefCounted {
     TouchEvent(WebCore::PageIdentifier pageID, WebCore::FrameIdentifier frameID, WebKit::WebTouchEvent event) -> (bool handled, struct std::optional<WebCore::RemoteUserInputEventData> remoteTouchEventData) MainThreadCallback
 #endif
 #if ENABLE(MAC_GESTURE_EVENTS)
-    GestureEvent(WebCore::FrameIdentifier frameID, WebCore::PageIdentifier pageID, WebKit::WebGestureEvent event) -> (std::optional<WebKit::WebEventType> eventType, bool handled, struct std::optional<WebCore::RemoteUserInputEventData> remoteUserInputEventData) MainThreadCallback
+    GestureEvent(WebCore::FrameIdentifier frameID, WebCore::PageIdentifier pageID, WebKit::WebGestureEvent event)
```

Comment:
Moving away from sendWithAsyncReply seems like a step in the wrong direction.  Can this really not be done with sendWithAsyncReply?

Reply:
I would need to pass the frameID or process into the lambda capture and get the connection that way. This matches what we do for other events, but I can change it if you think the former is better.

Reply:
I think we shouldn't move away from sendWithAsyncReply towards two different messages for something like this.